sdks/go/test/run_validatesrunner_tests.sh

```diff
@@ -262,22 +262,19 @@ elif [[ "$RUNNER" == "flink" || "$RUNNER" == "spark" || "$RUNNER" == "samza" ||
           --job-port $JOB_PORT \
           --expansion-port 0 \
           --artifact-port 0 &
-      SIMULTANEOUS=1
```

Comment:
I think you need to rebase your PR? Didn't you already merge this?

Reply:
Yep. I'll rebase again before I merge this PR.